keras_cv/models/stable_diffusion/stable_diffusion_test.py

```diff
@@ -12,30 +12,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import pytest
 import tensorflow as tf
 from tensorflow.keras import mixed_precision
 
 from keras_cv.models import StableDiffusion
 from keras_cv.tests.test_case import TestCase
 
 
+@pytest.mark.extra_large
 class StableDiffusionTest(TestCase):
-    def DISABLED_test_end_to_end_golden_value(self):
+    def test_end_to_end_golden_value(self):
         prompt = "a caterpillar smoking a hookah while sitting on a mushroom"
         stablediff = StableDiffusion(128, 128)
 
-        img = stablediff.text_to_image(prompt, seed=1337)
+        img = stablediff.text_to_image(prompt, seed=1337, num_steps=25)
```

Comment:
You could also just change the answer to `[32, 255, 238]`

Reply:
Yeah, I just wanted to make sure that something wasn't broken and we did in fact still have a stable forward pass. At the very least, this makes the test much faster because it does fewer diffusion steps.